credentials/credentials.go

```diff
@@ -54,9 +54,9 @@ var (
 	alpnProtoStr = []string{"h2"}
 )
 
-// Credentials defines the common interface all supported credentials must
+// PerRPCCredentials defines the common interface all supported per RPC credentials must
```

Comment:
PerRPCCredentials defines the common interface for the credentials which need to attach security information to every RPC (e.g., oauth2).